### PR TITLE
build: configure branch java-pubsub-lts as a release branch

### PR DESCRIPTION
**Pull request by hannahrogers-google**
enable releases

### CODE REVIEW

File: .github/release-please.yml [hunk 1]
```diff
@@ -1,2 +1,6 @@
 releaseType: java-yoshi
-bumpMinorPreMajor: true
\ No newline at end of file
+bumpMinorPreMajor: true
+branches:
+  - releaseType: java-lts
+    bumpMinorPreMajor: true
+    branch: java-pubsub-lts
```
Comment (stephaniewang526) [inline .github/release-please.yml:6]: I think we should avoid calling this `lts`, instead we should do something like `1.112.0-sp`

File: .github/sync-repo-settings.yaml [hunk 2]
```diff
@@ -1,49 +1,43 @@
...
+      - dependencies (11)
+      - linkage-monitor
+      - lint
+      - clirr
+      - units (7)
+      - units (8)
+      - units (11)
+      - 'Kokoro - Test: Integration'
+      - cla/google
+  - pattern: java-pubsub-lts
```
Comment (stephaniewang526) [inline .github/sync-repo-settings.yaml:21]: this needs to be updated if the above is updated.

**Review by stephaniewang526 — CHANGES_REQUESTED** [review]
please let me know if you have questions about the comments

File: .github/release-please.yml (hunk 1 above)
Comment (hannahrogers-google) [reply .github/release-please.yml:6]: Sounds good to me - I created a 1.111.0-sp branch instead.  Please see https://github.com/googleapis/java-pubsub/pull/640

**Comment by hannahrogers-google** [comment]
Opened https://github.com/googleapis/java-pubsub/pull/640 instead, please see this PR.

**Pull request closed without merging**